src/main/java/model/Lotto.java

```diff
@@ -0,0 +1,37 @@
+package model;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+public class Lotto {
+    private static final String LOTTO_NUMBERS_SIZE_MUST_BE_SIX = "로또 번호는 6개의 숫자여야 합니다.";
+    private static final String LOTTO_NUMBERS_MUST_BETWEEN_ONE_AND_FOURTYFIVE = "로또 번호는 1부터 45 사이의 숫자로 구성되어야 합니다.";
+    private static final String LOTTO_NUMBERS_MUST_NOT_OVERLAP = "로또 번호는 중복된 번호가 존재하면 안됩니다.";
+    public static final int PRICE = 1000;
+
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        validateNumbers(numbers);
+        this.numbers = numbers;
+    }
+
+    private void validateNumbers(List<Integer> numbers) {
+        if (numbers.size() != LottoNumbers.LOTTO_NUMBERS_SIZE) {
+            throw new IllegalArgumentException(LOTTO_NUMBERS_SIZE_MUST_BE_SIX);
+        }
+        if (numbers.stream().anyMatch(isBetweenOneAndFourtyFive.negate())) {
+            throw new IllegalArgumentException(LOTTO_NUMBERS_MUST_BETWEEN_ONE_AND_FOURTYFIVE);
+        }
+        if (numbers.stream().distinct().count() != LottoNumbers.LOTTO_NUMBERS_SIZE) {
```

Comment:
```suggestion
        int distinctNumbersLength = numbers.stream()
                .distinct()
                .count();
        if (distinctNumbersLength != LottoNumbers.LOTTO_NUMBERS_SIZE) {
```
로 변경하는 것이 더 좋을 것 같습니다. stream의 장점 중 하나가 가독성이 좋아진다는 것인데, 일렬로 쭉 늘여놓은 stream 구문은 그 장점을 보여주지 못하는 것 같습니다.

Reply:
놓친 부분인 것 같습니다. 피드백 감사합니다.